code/modules/atmospherics/machinery/components/unary_devices/bluespace_sender.dm

```diff
@@ -0,0 +1,201 @@
+/obj/machinery/atmospherics/components/unary/bluespace_sender
+	icon = 'icons/obj/atmospherics/components/bluespace_gas_selling.dmi'
+	icon_state = "bluespace_sender_off"
+	name = "Bluespace Gas Sender"
+	desc = "Sends gases to the bluespace network to be shared with the connected vendors, who knows what's beyond!"
+
+	density = TRUE
+	max_integrity = 300
+	armor = list(MELEE = 0, BULLET = 0, LASER = 0, ENERGY = 100, BOMB = 0, BIO = 100, RAD = 100, FIRE = 80, ACID = 30)
+	layer = OBJ_LAYER
+	circuit = /obj/item/circuitboard/machine/bluespace_sender
+	pipe_flags = PIPING_ONE_PER_TURF | PIPING_DEFAULT_LAYER_ONLY
+
+	///Base icon name for updating the appearance
+	var/base_icon = "bluespace_sender"
+	///Gas mixture containing the inserted gases and that is connected to the vendors
+	var/datum/gas_mixture/bluespace_network
+	///Rate of gas transfer inside the network (from 0 to 1)
+	var/gas_transfer_rate = 0.5
+	///A base price for each and every gases, in case you don't want to change them
+	var/list/base_prices = list(
+		/datum/gas/oxygen = 1,
+		/datum/gas/nitrogen = 1,
+		/datum/gas/carbon_dioxide = 1,
+		/datum/gas/miasma = 4,
+		/datum/gas/plasma = 1,
+		/datum/gas/nitrous_oxide = 1,
+		/datum/gas/bz = 2,
+		/datum/gas/hypernoblium = 5,
+		/datum/gas/water_vapor = 2,
+		/datum/gas/tritium = 3,
+		/datum/gas/stimulum = 5,
+		/datum/gas/nitryl = 4,
+		/datum/gas/pluoxium = 3,
+		/datum/gas/freon = 3,
+		/datum/gas/hydrogen = 2,
+		/datum/gas/healium = 4,
+		/datum/gas/proto_nitrate = 2,
+		/datum/gas/zauker = 10,
+		/datum/gas/helium = 1,
+		/datum/gas/antinoblium = 1,
+		/datum/gas/halon = 1
+	)
```

Comment:
As of #57844 being merged the gases have a value in their datum you can use, or allow players to set their own prices.

Reply:
reworked it to be an empty list that fills up on init with the current base prices of the various gases; players are still able to set their own or just keep the base prices